pages/index.tsx

```diff
@@ -88,13 +91,18 @@ const Home: NextPage<Props> = ({officers}) => {
   );
 };
 
-export async function getStaticProps(context:NextPageContext) {
-  let offficerQuery: Officer = {chapter: 'national'}
-  let officers: Officer[] = await getOfficers(offficerQuery)
+export async function getStaticProps() {
+  let officerQuery: Officer = {chapter: 'national'}
+  let officers: Officer[] = await getOfficers(officerQuery)
+
+  // Get all chapters. Filter by region in component once user's location is retrieved.
+  // Navigator is undefined in async getStaticProps(), so must do it in component.
+  let chapters: Chapter[] = await getChapters({})
 
   return {
     props: {
-      officers: JSON.parse(JSON.stringify(officers))
+      officers: JSON.parse(JSON.stringify(officers)),
+      chapters: JSON.parse(JSON.stringify(chapters)),
```

Comment:
Is this necessary or can we just do
```
props: {
  officers: officers,
  chapters: chapters,
}
```

Reply:
For some reason it isn't serializable unless you do this. I'm thinking that the ObjectID on these types are the cause, but this should be ok